mona_sdk/mona_single_message.py

```diff
@@ -47,17 +58,24 @@ class MonaSingleMessage:
     sendTimestamp: int or str = None
 
     def __post_init__(self):
-        if self.exportTimestamp is not None and self.sendTimestamp is None:
+        if not MONA_SDK_USE_SEND_TIMESTAMP_FIELD:
```

Comment:
nit - switch to if - else instead of if not - else
you can switch the env to be MONA_SDK_USE_OLD_EXPORT_TIMESTAMP_FIELD